src/db/migrations/50-adjust-relics-data.ts

```diff
@@ -0,0 +1,54 @@
+import { Knex } from 'knex';
+import _ from 'lodash';
+
+import { RELICS_SEASON_1_FACTORY } from '../../constants/artistIntegrations';
+import { CrdtOperation } from '../../types/message';
+import { Table } from '../db';
+
+export const up = async (knex: Knex) => {
+  console.log('Removing RELICS season 1 data for reindexing');
+  const tracks = await knex(Table.processedTracks).where({ platformId: 'relics' })
+  console.log('# of tracks:');
+  console.log(tracks.length);
+
+  const artworkCids = _.compact(_.uniq(tracks.map((track) => track.lossyArtworkIPFSHash)));
+  const artworkUrls = _.compact(_.uniq(tracks.map((track) => track.lossyArtworkUrl)));
+
+  console.log('# of cids:');
+  console.log(artworkCids.length);
+  console.log('# of urls:');
+  console.log(artworkUrls.length);
+
+  // remove relics tracks
+  await knex(Table.processedTracks).where({ platformId: 'relics' }).del()
+
+  // remove nftsProcessedTracks join
+  await knex(Table.nfts_processedTracks).whereIn('processedTrackId', tracks.map((track) => track.id)).del()
+
+  // remove ipfsFiles
+  if (artworkCids.length > 0) {
+    await knex(Table.ipfsFiles).whereIn('cid', artworkCids).del()
+  }
+
+  // remove ipfsPins
+  if (artworkUrls.length > 0) {
+    await knex(Table.ipfsPins).whereIn('id', artworkCids).del()
+  }
+
+  // remove ipfsFilesUrls
+  if (artworkUrls.length > 0) {
+    await knex(Table.ipfsFilesUrls).whereIn('url', artworkUrls).del()
+  }
```

Comment:
seems unnecessary?

Reply:
Gets re-added as required but I can remove.

Reply:
all 3 ipfs sections can be removed right?